auth-api/migrations/versions/3c89ccb6bb32_actor_changed_to_actor_id.py

```diff
@@ -0,0 +1,26 @@
+"""actor changed to actor_id
+
+Revision ID: 3c89ccb6bb32
+Revises: d0392ebda924
+Create Date: 2021-05-03 05:29:19.193588
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '3c89ccb6bb32'
+down_revision = 'd0392ebda924'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_column('activity_logs', 'actor')
+    op.add_column('activity_logs', sa.Column('actor_id', sa.Integer(), nullable=True))
```

Comment:
foreign_key to user?

Reply:
I am not changing this now..but will keep this in mind and see in next sprint as well if we can do that.